gammapy/maps/core.py

```diff
@@ -1129,7 +1135,17 @@ def plot_interactive(self, rc_params=None, **kwargs):
         interact_kwargs = {}
 
         for axis in self.geom.axes:
-            options = axis.as_plot_labels
+            if axis.node_type == "center":
+                if axis.name == "energy":
+                    options =  energy_unit_format(axis.center)
+                else:
+                    options = axis.as_plot_labels
+            else:
+                if axis.name == "energy":
+                    E = energy_unit_format(axis.edges)
+                    options = [f"{E[i]} - {E[i+1]}" for i in range(len(E)-1)]
```

Comment:
Using ``itertools.pairwise``

```suggestion
                    options = [f"{e1} - {e2}" for e1, e2 in pairwise(E)]
```